### PR TITLE
Simplify pyaction/pyaction_wconprod_insert_kw.py script

### PR DESCRIPTION
**Pull request by lisajulia**
Simplified script, still produces the same output

### CODE REVIEW

**Comment by lisajulia** [comment]
@blattms: Can you have another look here? Thanks!

**Review by blattms — COMMENTED** [review]
So basically globals() and locals() are the same here as we are at module level?

File: pyaction/pyaction_wconprod_insert_kw.py [hunk 1]
```diff
@@ -1,84 +1,55 @@
...
+    all_wells_shut = False
+    setup_done = True
 
-    current_time = schedule.start + datetime.timedelta(seconds=summary_state.elapsed())
+current_time = schedule.start + datetime.timedelta(seconds=summary_state.elapsed())
 
-    act01_days_since_last_execution = (current_time - globals()["act01_last_executed_at"]).total_seconds()/3600/24  
-    act02_days_since_last_execution = (current_time - globals()["act02_last_executed_at"]).total_seconds()/3600/24  
-    act03_days_since_last_execution = (current_time - globals()["act03_last_executed_at"]).total_seconds()/3600/24  
+days_since_last_update = [(current_time - last_updated_at[0]).total_seconds()/3600/24, (current_time - last_updated_at[1]).total_seconds()/3600/24, (current_time - last_updated_at[2]).total_seconds()/3600/24]
```
Comment (blattms) [inline pyaction/pyaction_wconprod_insert_kw.py:19]: Newbee question :sweat_smile:: Would this also work?
```suggestion
days_since_last_update = [(current_time - i).total_seconds()/3600/24 for i in last_updated_at]
```

**Comment by lisajulia** [comment]
> So basically globals() and locals() are the same here as we are at module level?

Yes, here they are, yet inside the run function, I had to use globals, because otherwise the variables would have been only at local scope.

**Comment by lisajulia** [comment]
jenkins build this please

**Pull request merged**